deploy/kubernetes/dolphinscheduler/Chart.yaml

```diff
@@ -35,11 +35,11 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 3.1.0
+version: latest
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 3.1.0
+appVersion: latest
```

Comment:
I am not sure whether the latest tag work here or not, I think there are some cases chart.yaml in dev branch not work in our latest version?  @kezhenxu94 do you have time to take a look?